src/index.html

```diff
@@ -7,8 +7,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav__container">
+        <div class="logo__container">
+          <a href="#home">
+            <img src="/images/logo.png" alt="MOYO">
+          </a>
+        </div>
+        <ul class="nav__list">
+          <li><a href="#apple" class="nav_item is-active">Apple</a></li>
+          <li><a href="#samsung" class="nav_item">Samsung</a></li>
+          <li><a href="#smartphones" class="nav_item">Smartphones</a></li>
+          <li>
+            <a
+              href="#laptops&computers"
+              class="nav_item"
+              data-qa="hover"
+            >Laptops & computers</a>
```

Comment:
see [this message](https://mateacademy.slack.com/archives/C050SHYFFR9/p1681912970291919)